artman/cli/main.py

```diff
@@ -277,6 +275,40 @@ def normalize_flags(flags, user_config):
             api=flags.api,
             googleapis=googleapis,
         )
+    elif flags.batch:
+        googleapis = os.path.realpath(os.path.expanduser(
+            pipeline_args['local_paths']['googleapis'],
+        ))
+        flags.config = '{googleapis}/gapic/batch/common.yaml'.format(
+            googleapis=googleapis,
+        )
+        default_pipeline = 'GapicClientBatchPipeline'
+        if not flags.publish:
+            # If publish flag was not set by the user, set it here.
+            # This prevents the user config yaml from causing a
+            # publish event when batch mode is used.
+            flags.publish = 'noop'
+        if flags.target:
```

Comment:
The repo is still taken from the config YAML file when `--target` is used; it just changes _which_ repo. (That said, I agree that batch mode should never be used to publish anywhere other than api-client-staging and api-client-staging-private.)